evo-delivery-app/evo-delivery-frontend/src/index.tsx

```diff
@@ -45,7 +45,7 @@ root.render(
           <Route path={ENTITIES.drivers}>
             <Route index element={<Drivers />} />
             <Route path={`${ENTITY_VIEW_STATES.create}`} element={<CreateDriver />} />
-            <Route path={`:driverId/${ENTITY_VIEW_STATES.view}`} element={<Driver />} />
+            <Route path={`:driverId/${ENTITY_VIEW_STATES.view}`} element={<ViewDriver />} />
```

Comment:
Lets stick to the same naming conventions.
If you decide to change this component name, it's ok, but please change it for all entities..